<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemperatureViewer</class>
 <widget class="QDialog" name="TemperatureViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Temperatures Viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Choose a study</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Study</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTempFile"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonBrowse">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Processed measures</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Shaft</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pressure Sensor</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Study</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Labo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Raw temperatures</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Raw pressures</string>
       </property>
      </item>
     </widget>
     <widget class="QTableView" name="tableView"/>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TemperatureViewer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TemperatureViewer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
